src/script/cpp_api/s_security.cpp

```diff
@@ -358,6 +367,13 @@ void ScriptApiSecurity::initializeSecurityClient()
 	copy_safe(L, os_whitelist, sizeof(os_whitelist));
 	lua_setfield(L, -3, "os");
 	lua_pop(L, 1);  // Pop old OS
+	
+	// Copy safe OS functions
```

Comment:
these functions are absolutely not safe and allow unsandboxed access to the client filesystem

Reply:
Will remove this soon